src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
```

Comment:
Health isn't passed in the constructor so you can remove it from the parameters.

Reply:
agree, fix @Anastasiia-Khudych comment, health value won't change and will be the same for all animals